src/lib.rs

```diff
@@ -1,5 +1,262 @@
 //! The utility functions and data structures of this project.
 
+use std::path::PathBuf;
+
+use clap::{Parser, Subcommand};
+
+/// Parses the Git history.
+#[derive(Parser, Debug)]
+#[clap(version, about, long_about = None)]
+pub struct Args {
+    #[clap(subcommand)]
+    input_method: InputMethod,
+
+    /// Always show the entire output.
+    #[clap(short = 'v', long = "verbose")]
+    is_verbose: bool,
```

Comment:
https://github.com/kevinmatthes/cargo-optimise uses different levels of verbosity.  Would this work for this case, as well?

Reply:
Okay, it was just a suggestion since the topics are related.